.github/workflows/linting.yml

```diff
@@ -522,3 +528,34 @@ jobs:
 
       - run: pnpm install
       - run: .github/files/lint-project-structure.sh
+
+  ### Warn on manual updates to Phan auto-generated stubs
+  phan_stubs:
+    name: Phan stubs
+    runs-on: ubuntu-latest
+    needs: changed_files
+    if: needs.changed_files.outputs.phanstubs == 'true' && github.event.pull_request.user.login != 'matticbot'
+    timeout-minutes: 5  # 2025-07-28: Probably takes about a minute.
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/deepen-to-merge-base
+        id: deepen
+      - name: Warn about stubs
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          BASE: ${{ steps.deepen.outputs.merge-base }}
+          HEAD: ${{ github.event.pull_request.head.sha }}
+        run: |
+          COMMENTS='[]'
+          while IFS= read -r FILE; do
+            # TODO: If they add a way to do file-level comments in this API, switch to that.
+            COMMENTS=$( jq --arg file "$FILE" '. += [ { path: $file, body: "This file is automatically generated and should not be edited directly. See the comment at the top of the file for details.\n\nIf you have any questions, ping `@jetpack-monorepo`.", position: 1 } ]' <<<"$COMMENTS" )
```

Comment:
The linked PR looks good. I do wonder if it needs to be @Automattic/jetpack-monorepo instead of @jetpack-monorepo (testing in this comment).

Reply:
Looks like only the former triggers a notice.